exporter/importables.go

```diff
@@ -1971,4 +2022,75 @@ var resourcesMap map[string]importable = map[string]importable{
 			// {Path: "http_url_spec.authorization", Variable: true},
 		},
 	},
+	"databricks_access_control_rule_set": {
+		AccountLevel: true,
+		Service:      "access",
+		// Name: func(ic *importContext, d *schema.ResourceData) string {
```

Comment:
we don't require `Name` for access_control?

Reply:
No, it's not strictly required - if the `Name` function isn't defined, then a default name is generated